src/edit/common/LocationSelectionWithCheck.test.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  screen,
+  act,
+} from '@folio/jest-config-stripes/testing-library/react';
+import { fireEvent } from '@folio/jest-config-stripes/testing-library/dom';
+import { LocationSelectionWithCheck } from './LocationSelectionWithCheck';
+import '../../../test/jest/__mock__';
+import { renderWithFinalForm, renderWithIntl } from '../../../test/jest/helpers';
+import renderWithRouter from '../../../test/jest/helpers/renderWithRouter';
+
+jest.mock('../../RemoteStorageService', () => ({
+  Check: {
+    useByLocation: jest.fn(() => jest.fn(() => true)), // Mocking always true for check
+  },
+  Confirmation: {
+    Heading: jest.fn(() => 'Remote Storage Heading'),
+    Message: jest.fn(() => 'Remote Storage Message'),
+  },
+}));
+
+jest.mock('@folio/stripes/smart-components', () => ({ ...jest.requireActual('@folio/stripes/smart-components'),
```

Comment:
Please format this block so:
1. each item is on a new line
2. there is a trailing coma after each item
3. each jsx prop is on a new line

Reply:
Done!